app.py

```diff
@@ -0,0 +1,142 @@
+from datetime import datetime
+from flask import Flask, render_template, request, request, jsonify
+from dotenv import load_dotenv
+
+#models
+from Infrastructure.domain.entities import models
+
+import os
+import requests
+import json
+
+load_dotenv()
+app = Flask(__name__)
+
+AUTHORIZE_URL = os.getenv("RSO_BASE_URI") + "/authorize"
+TOKEN_URL = os.getenv("RSO_BASE_URI") + "/token"
+LOGOUT_URL = os.getenv("RSO_BASE_URI") + "/logout"
+APP_CALLBACK_URL = os.getenv("APP_BASE_URL") +":"+ os.getenv("PORT") + os.getenv("APP_CALLBACK_PATH")
+RSO_CLIENT_ID = os.getenv("RSO_CLIENT_ID")
+AUTH_URL=AUTHORIZE_URL + "?redirect_uri=" + APP_CALLBACK_URL + "&client_id=" + RSO_CLIENT_ID + "&response_type=code" + "&scope=openid"
+token = ""
+
+def set_token(value: str):
+    global token
```

Comment:
I'd rather avoid globals, can we put this on a shared object on the `app` instead or something? Maybe this thing: https://flask.palletsprojects.com/en/1.0.x/api/#flask.g

Reply:
using session flask instead